examples/client/client.c

```diff
@@ -358,6 +360,59 @@ static void SetKeyShare(WOLFSSL* ssl, int onlyKeyShare, int useX25519,
         } while (ret == WC_PENDING_E);
     #endif
     }
+    #ifdef HAVE_LIBOQS
+    if (onlyKeyShare == 3) {
+        if (useLibOqs) {
+            int group = 0;
+
+            if (XSTRNCMP(oqsAlg, "KYBER512", XSTRLEN("KYBER512")) == 0)
+                group = WOLFSSL_KYBER512;
+            else if(XSTRNCMP(oqsAlg, "KYBER768",
+                                XSTRLEN("KYBER768")) == 0)
+                group = WOLFSSL_KYBER768;
+            else if(XSTRNCMP(oqsAlg, "KYBER1024",
+                                XSTRLEN("KYBER1024")) == 0)
+                group = WOLFSSL_KYBER1024;
+            else if(XSTRNCMP(oqsAlg, "NTRU_HPS2048509",
+                                XSTRLEN("NTRU_HPS2048509")) == 0)
+                group = WOLFSSL_NTRU_HPS2048509;
+            else if(XSTRNCMP(oqsAlg, "NTRU_HPS2048677",
+                                XSTRLEN("NTRU_HPS2048677")) == 0)
+                group = WOLFSSL_NTRU_HPS2048677;
+            else if(XSTRNCMP(oqsAlg, "NTRU_HPS4096821",
+                                XSTRLEN("NTRU_HPS4096821")) == 0)
+                group = WOLFSSL_NTRU_HPS4096821;
+            else if(XSTRNCMP(oqsAlg, "NTRU_HRSS701",
+                                XSTRLEN("NTRU_HRSS701")) == 0)
+                group = WOLFSSL_NTRU_HRSS701;
+            else if(XSTRNCMP(oqsAlg, "LIGHTSABER",
+                                XSTRLEN("LIGHTSABER")) == 0)
+                group = WOLFSSL_LIGHTSABER;
+            else if(XSTRNCMP(oqsAlg, "SABER",
+                                XSTRLEN("SABER")) == 0)
+                group = WOLFSSL_SABER;
+            else if(XSTRNCMP(oqsAlg, "FIRESABER",
+                                XSTRLEN("FIRESABER")) == 0)
+                group = WOLFSSL_FIRESABER;
+            else if(XSTRNCMP(oqsAlg, "KYBER90S512",
+                                XSTRLEN("KYBER90S512")) == 0)
+                group = WOLFSSL_KYBER90S512;
+            else if(XSTRNCMP(oqsAlg, "KYBER90S768",
+                                XSTRLEN("KYBER90S768")) == 0)
+                group = WOLFSSL_KYBER90S768;
+            else if(XSTRNCMP(oqsAlg, "KYBER90S1024",
+                                XSTRLEN("KYBER90S1024")) == 0)
+                group = WOLFSSL_KYBER90S1024;
+            else
```

Comment:
Seems redundant, since group is set to zero above. Also can you add some braces around these?

Reply:
Will remove `else` clause and  yes, I will put braces.